resources.tf

```diff
@@ -277,7 +277,7 @@ output "analytical_dataset_generation" {
 
 resource "aws_acm_certificate" "analytical_dataset_generation" {
   certificate_authority_arn = data.terraform_remote_state.aws_certificate_authority.outputs.cert_authority.arn
-  domain_name               = "analytical_dataset_generation.${local.env_prefix[local.environment]}dataworks.dwp.gov.uk"
+  domain_name               = "analytical.dataset.generation.${local.env_prefix[local.environment]}dataworks.dwp.gov.uk"
```

Comment:
I think you will also find `dataworks.dwp.gov.uk` in your TF locals

Reply:
Dan , Don't have dataworks.dwp.gov.uk in TF locals

Reply:
I don't like that FQDN. dots are special and denote subdomains. It should go back to the original name.

The domain name part of the FQDN can be grabbed from parameter store as `dataworks_domain_name`. So if you could update the `terraform.tf.j2` file to pull that in then you should be able to reference it as another local.

To be honest, `env_prefix` could do with being hoisted up to that parameter too, but that's too much for this ticket, so leave it like this for now and we'll see about that when we migrate to secrets manager.

Reply:
I have now reference it via a TF locals, can you please recheck now.